app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -444,6 +447,19 @@ def mail_delivery_params(proofed_at)
     return { wait_until: wait_until, queue: :intentionally_delayed }
   end
 
+  def send_enrollment_status_sms_notification(enrollment:)
+    return unless IdentityConfig.store.in_person_send_proofing_notifications_enabled
+    return if enrollment&.proofed_at.blank?
+    sms_delay_hours = IdentityConfig.store.in_person_results_delay_in_hours ||
+                      DEFAULT_EMAIL_DELAY_IN_HOURS
+    wait_until = enrollment.proofed_at + sms_delay_hours
```

Comment:
`enrollment.proofed_at` may not be the correct datetime because of a bug in the USPS system. See LG-10248 for more info. For now we should just wait 1 hour from when we get the status update from USPS

```suggestion
    wait_until = Time.zone.now + sms_delay_hours
```

Reply:
@sheldon-b , i am aware of that, that card is in the same sprint, if that's fixed, this will be correct, so it may be good for now to be consistent with the email behavior?

Reply:
Sure that's fine with me. I'll update LG-10248 to mention that it should update the SMS behavior as well